ssm/ssm2.py

```diff
@@ -343,10 +345,27 @@ def _save_msg_to_queue(self, body, empaid):
                 name = self._inq.add({'body': extracted_msg,
                                       'signer': signer,
                                       'empaid': empaid})
-                log.info("Message saved to incoming queue as %s", name)
+                
+                log.info("Message saved to incoming queue as %s", name)     
```

Comment:
```suggestion
                log.info("Message saved to incoming queue as %s", name)
```
some trailing whitespace here :)